weblate/templates/keyboard-shortcuts.html

```diff
@@ -0,0 +1,166 @@
+{% load i18n %}
+
+<!-- Modal that shows available keyboard shortcuts  -->
+<div class="modal fade " id="shortcuts-modal" tabindex="-1" role="dialog" aria-labelledby="shortcutsModal">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{% trans " Close" %}">
```

Comment:
Remove extra whitespace:

```suggestion
        <button type="button" class="close" data-dismiss="modal" aria-label="{% trans "Close" %}">
```